ldms/src/core/ldms_xprt.c

```diff
@@ -375,6 +375,7 @@ static void send_dir_update(struct ldms_xprt *x,
 	reply->hdr.xid = x->remote_dir_xid;
 	reply->hdr.cmd = htonl(LDMS_CMD_DIR_UPDATE_REPLY);
 	reply->hdr.rc = 0;
+	reply->dir.more = 0;
```

Comment:
It doesn't hurt to set the flag to 0, but upon code inspection the `more` flag is ignored in the `DIR_UPD` message.

Reply:
yes. this is an information leak patch, not a "functional" change. Thank you for confirming that more is unused at the other end; I thought so but it's good to have a second opinion on network code.